.github/workflows/flow-deploy-release-artifact.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
         with:
           action: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
           with: |
-            node-version: 20
+            node-version: 18.19.0
```

Comment:
Really we should move to Node 20.x since Node 18.x will be deprecated later this year.

Reply:
Changed to 20.14.0 LTS